README.md

```diff
@@ -2,10 +2,34 @@
 
 Active development is currently happening at [fivejars/openy_gated_content](https://github.com/fivejars/openy_gated_content). Pull requests should be submitted there. They will be pulled into this repo as they are reviewed and released.
 
-## Development
+## Installation
+
+See also the [video version of these instructions](https://youtu.be/vlqv4ly3iak). This assumes you've already [built an Open Y site](https://github.com/ymcatwincities/openy-project#installation) and have it
+ running.
+
+- Add this module to your codebase.
+  - via composer: `composer require ymcatwincities/openy_gated_content`
+  - without composer: this is not recommended.
+- Enable the modules
+   - Through the UI:
+     - Visit **Extend** in your toolbar.
+     - Check "Virtual Y Base", "Open Y Virtual YMCA Storage", and "Open Y
+      Virtual YMCA Auth Example".
+      - Install and say "yes" to add all required dependencies.
+   - Via drush: `drush en openy_gated_content openy_gc_storage
+    openy_gc_auth_example -y`
+- Add at least one term in these vocabularies via **Structure** > **Taxonomies**
+  - Virtual Y Category
+  - Virtual Y Equipment
+  - Virtual Y Level
+- Create a **Landing Page** and add the **Virtual Y Content** paragraph to the
```

Comment:
After the user segmentation merge we also need a page with a login paragraph and steps for a setup these pages URLs in settings.

cc @froboy @anpolimus

Reply:
@hamrant I'll add it in my next PR.
Lets merge it for now